docs/TROUBLESHOOTING.md

```diff
@@ -74,9 +74,9 @@ $ oc adm node-logs -l kubernetes.io/os=windows --path=/containerd/containerd.log
 ## How to collect Windows application event logs
 
 The Get-WinEvent shim on the kubelet logs endpoint can be used to collect application event logs from Windows machines.
-E.g. for the docker runtime service:
+E.g. for the containerd runtime service:
 ```shell script
-$ oc adm node-logs -l kubernetes.io/os=windows --path=journal -u docker
+$ oc adm node-logs -l kubernetes.io/os=windows --path=journal -u containerd
```

Comment:
This won't work. See  [How to collect containerd runtime logs](https://github.com/openshift/windows-machine-config-operator/pull/1052/files#diff-96160ca341eafe20466fcf7f74d2d720f3f05a614452c5ab41f32d8756a3fec0R71)